### ISSUE TITLE
Add option to disable e-mail notifications for Agents

### ISSUE THREAD

**Issue by michnovka** [chatwoot/chatwoot#8944]
### Is your feature or enhancement related to a problem? Please describe.

I have 2 reasons for this: 

1. for security reasons, we do not want our employees to get e-mails automatically with the whole transcript. of course they can copy each chat, screenshot it, whatever, but its one thing to do that, another thing to work in a company and get hundreds of emails, then leave as a disgruntled ex-employee with the database of company clients in your inbox (and potentially other sensitive data)
2. many of our CSRs do not even have access to mailbox, so even if we at least had the option to disable email notifications by default per user, and allow them to enable it later (now some email notifications are enabled by default for every new agent), it makes no sense and creates tons of "email not delivered" errors on our SMTP server

### Describe the solution you'd like

I would like an option per user, configurable by Admins, to disable any kinds of e-mail notifications. In such case the https://chatwoot.app/app/accounts/1/profile/settings would have all the checkboxes under e-mail notifications disabled.

**Comment by masteradit** [comment]
Hi, any updates on this?

**Comment by michnovka** [comment]
Hi, any update on this?

**Comment by pauleke65** [comment]
<img width="931" alt="Image" src="https://github.com/user-attachments/assets/508e2ae1-74b6-4fc0-822a-d820eb6ac358" />

Not a fix, but a work-around for if you're self hosting, you could write a script to set the `email_flags` for the invited user to `0`. How you'd trigger the script is another issue tho.

**Comment by michnovka** [comment]
@pauleke65 interesting, so there is an option for this in DB? Do you know what other values mean?

If this is a matter of UI only, then we could have this done fairly easily, no?

**Comment by pauleke65** [comment]
@michnovka not really, but I think they're set to the number of email options enabled, you can try playing with the values and see how it changes in the db

**Comment by brunocardosobc** [comment]
> [@pauleke65](https://github.com/pauleke65) interesting, so there is an option for this in DB? Do you know what other values mean?
> 
> If this is a matter of UI only, then we could have this done fairly easily, no?

I believe that is the index of each label/value on this const.

export const NOTIFICATION_TYPES = [
  { // 1
    label: 'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.CONVERSATION_CREATED',
    value: 'conversation_creation',
  },
  { // 2
    label: 'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.CONVERSATION_ASSIGNED',
    value: 'conversation_assignment',
  },
  { // 3
    label: 'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.CONVERSATION_MENTION',
    value: 'conversation_mention',
  },
  { // 4
    label:
      'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.ASSIGNED_CONVERSATION_NEW_MESSAGE',
    value: 'assigned_conversation_new_message',
  },
  { // 5
    label:
      'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.PARTICIPATING_CONVERSATION_NEW_MESSAGE',
    value: 'participating_conversation_new_message',
  },
  { // 6
    label:
      'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.SLA_MISSED_FIRST_RESPONSE',
    value: 'sla_missed_first_response',
  },
  { // 7
    label: 'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.SLA_MISSED_NEXT_RESPONSE',
    value: 'sla_missed_next_response',
  },
  { // 8
    label: 'PROFILE_SETTINGS.FORM.NOTIFICATIONS.TYPES.SLA_MISSED_RESOLUTION',
    value: 'sla_missed_resolution',
  },
];

**Comment by michnovka** [comment]
I just found out we have 6GB of "Undelivered Mail Returned to Sender" emails, when the chat transcript is attempted to be sent to CSR email (which are dummy emails as they dont use emails at our company). Surely there must be some way to disable this!
